run-dask-process.sh

```diff
@@ -26,11 +26,12 @@ ARGS=$*
 function hasArg {
     (( ${NUMARGS} != 0 )) && (echo " ${ARGS} " | grep -q " $1 ")
 }
-VALIDARGS="-h --help scheduler workers"
+VALIDARGS="-h --help scheduler workers tcp ucx ucxib ucx-ib"
 HELP="$0 [<app> ...] [<flag> ...]
  where <app> is:
-   scheduler        - start dask scheduler
-   workers          - start dask workers
+   scheduler               - start dask scheduler
+   workers                 - start dask workers
+   tcp/ucx/ucxib/ucx-ib    - communication type for cluster
```

Comment:
This isn't required but I've often followed the convention of "options" vs. "targets" for these, where options start with `--`.  For this, you could say `--tcp`, etc. are all options for how to start the workers and scheduler targets.

Reply:
Agreed. Updated to have these as flags rather than targets. Let me know if the change look good and I'll merge.